both/methods.js

```diff
@@ -153,16 +153,25 @@ Meteor.methods({
 
     Meteor.call('refreshRoundCodes', competitionId, eventCode);
   },
-  addNonEventRound: function(competitionId, round) {
+  addScheduleEvent: function(competitionId, eventData, roundId) {
     check(competitionId, String);
     throwIfCannotManageCompetition(this.userId, competitionId);
-    return Rounds.insert({
+
+    var round = Rounds.findOne(roundId);
+
+    return ScheduleEvents.insert({
       competitionId: competitionId,
-      title: round.title,
-      startMinutes: round.startMinutes,
-      durationMinutes: round.durationMinutes,
+      roundId: (round ? round._id : null),
```

Comment:
Backtrace might be useless here as there probably is never any interesting code in the stack above us (just meteor library code). Still, knowing which of our myriad `console.log(err)` is the one that fired seems useful.